core/build.sbt

```diff
@@ -276,9 +276,6 @@ lazy val plugin =
 
       buildInfoKeys := Seq[BuildInfoKey](version),
       buildInfoPackage := "cloudflow.sbt",
-
-      resolvers += Resolver.url("lightbend-oss", url("https://lightbend.bintray.com/cloudflow"))(Resolver.ivyStylePatterns),
```

Comment:
I guess that we need to wait for Sonatype to test this, right?

Reply:
I didnt see any issues building things locally.

Reply:
I removed this in my PR https://github.com/lightbend/cloudflow/pull/9 .. @skonto did u test removing artifacts from your local repo ? Just to check if it's being actually pulled from remote ?

Reply:
yes I removed everything from both maven and ivy.

Reply:
👍